src/Author.java

```diff
@@ -0,0 +1,70 @@
+import java.util.Objects;
+
+public class Author {
+    private String firstName;
+    private String middleName;
+    private String lastName;
+
+    public Author(String firstName, String middleName, String lastName) {
+        this.firstName = firstName;
+        this.middleName = middleName;
+        this.lastName = lastName;
+    }
+
+    public void setFirstName(String firstName) {
+        this.firstName = firstName;
+    }
+
+    public void setMiddleName(String middleName) {
+        this.middleName = middleName;
+    }
+
+    public void setLastName(String lastName) {
+        this.lastName = lastName;
+    }
+
+    public String getLastName() {
+        return this.lastName;
+    }
+
+    public String getMiddleName() {
+        return this.middleName;
+    }
+
+    public String getFirstName() {
+        return this.firstName;
+    }
+
+
+    public String toString() {
+        String authorToString = this.getFirstName() + " " + this.getMiddleName() + " " + this.getLastName();
+        return authorToString;
+    }
+
+    public boolean equals(Object author2) {
+
+        if(this.getClass() != author2.getClass())
```

Comment:
```suggestion
        if(author2 == null || this.getClass() != author2.getClass())
```
по контракту метода equals нужно возвращать false, если придёт null. У тебя сейчас будет просто исключение